CI-app - Swift -  Document instrumenting stdout and stderr (#12822)
DataDog/documentation: content/en/continuous_integration/setup_tests/swift.md

For the following configuration settings:
 - `Boolean` variables can use any of: `1`, `0`, `true`, `false`, `YES`, or `NO`
 - `String` list variables accept a list of elements separated by `,` or `;`
 
### Enabling auto-instrumentation

`DD_ENABLE_STDOUT_INSTRUMENTATION`

> : Captures messages written to `stdout` (e.g `print()` ) and reports them as Logs. (Implies charges for Logs product) (Boolean)

Reviewer: ```suggestion
: Captures messages written to `stdout` (for example, `print()` ) and reports them as logs. This may impact your bill. (Boolean)
```

> : Captures messages written to `stdout` (for example, `print()` ) and reports them as logs. This may impact your bill. (Boolean)

Reviewer: ```suggestion
: Captures messages written to `stdout` (for example, `print()`) and reports them as logs. This may impact your bill. (Boolean)
```

`DD_ENABLE_STDERR_INSTRUMENTATION`

> : Captures messages written to `stderr` (e.g `NSLog()`, UITest steps ) and reports them as Logs. (Implies charges for Logs product) (Boolean)

Reviewer: ```suggestion
: Captures messages written to `stderr` (for example, `NSLog()`, UITest steps ) and reports them as logs. This may impact your bill. (Boolean)
```

> : Captures messages written to `stderr` (for example, `NSLog()`, UITest steps ) and reports them as logs. This may impact your bill. (Boolean)

Reviewer: ```suggestion
: Captures messages written to `stderr` (for example, `NSLog()`, UITest steps) and reports them as logs. This may impact your bill. (Boolean)
```
